ui/lib/apps/SlowQuery/translations/en.yaml

```diff
@@ -34,22 +34,46 @@ slow_query:
     db: Execution Database
     db_tooltip: The database used to execute the query
 
-    parse_time: Parse Time
+    query_time2: Query Time
+    query_time2_tooltip: The execution time of a query (due to the parallel execution, it may be significantly smaller than the following time)
+    parse_time: 　　Parse Time
     parse_time_tooltip: Time consumed when parsing the query
-    compile_time: Compile Time
-    compile_time_tooltip: Time consumed when optimizing the query
-    wait_time: Coprocessor Wait Time
-    process_time: Coprocessor Process Time
-    backoff_time: Backoff Retry Time
+    compile_time: 　　Generate plan Time
+    compile_time_tooltip: Time consumed when generating plan of the query
+    rewrite_time: 　　　　Rewrite plan Time
+    preproc_subqueries_time: 　　　　　　Preprocess subquery Time
+    preproc_subqueries_time_tooltip: The time consumed of pre-process the subquery during the rewrite plan phase
+    optimize_time: 　　　　Optimize plan Time
+    wait_ts: 　　Wait Txn StartTS Time
+    wait_ts_tooltip: Time consumed of waiting transaction start ts
```

Comment:
wait for retrieving a StartTs finish?

Since retrieving StartTs is asynchronous, when we need the `startTs`, we need wait for retrieving a StartTs finish first.